src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js

```diff
@@ -62,16 +64,44 @@ const ActivityThread = ({
     repliesTotalCount = 0,
```

Comment:
What happens if this is defaulted to 0? Can we add tests for this edge case

Reply:
It's 0 when there are no replies. In that case the button is not rendered. I've updated tests to cover that scenario.